docs/operations/auth-ldap.md

```diff
@@ -22,182 +23,279 @@ title: "LDAP auth"
-    "roles": [
-        "readRole"
-    ]
+    "name": "group1map",
+    "groupPattern": "CN=group1,CN=Users,DC=example,DC=com",
+    "roles": [
+        "readRole"
+    ]
 }
 ```
 
-You can configure the mapping as follows:
+The following example request configures the mapping&mdash;the role mapping is in the file `groupmap.json`. See [Configure Druid for LDAP authentication](#configure-druid-for-ldap-authentication) for the contents of an example file.
 
 ```
-curl -i -v  -H "Content-Type: application/json" -u internal -X POST -d @groupmap.json http://localhost:8081/druid-ext/basic-security/authorization/db/ldapauth/groupMappings/group1map
+curl -i -v  -H "Content-Type: application/json" -u internal -X POST -d @groupmap.json http://localhost:8081/druid-ext/basic-security/authorization/db/ldapauth/groupMappings/group1map
 ```
 
-To check whether the group mapping was created successfully, run the following command:
+To check whether the group mapping was created successfully, the following request lists all group mappings:
 
 ```
-curl -i -v  -H "Content-Type: application/json" -u internal -X GET http://localhost:8081/druid-ext/basic-security/authorization/db/ldapauth/groupMappings
+curl -i -v  -H "Content-Type: application/json" -u internal -X GET http://localhost:8081/druid-ext/basic-security/authorization/db/ldapauth/groupMappings
 ```
 
-To check the details of a specific group mapping, use the following:
+The following example request returns the details of the `group1map` group:
 
 ```
-curl -i -v  -H "Content-Type: application/json" -u internal -X GET http://localhost:8081/druid-ext/basic-security/authorization/db/ldapauth/groupMappings/group1map
+curl -i -v  -H "Content-Type: application/json" -u internal -X GET http://localhost:8081/druid-ext/basic-security/authorization/db/ldapauth/groupMappings/group1map
 ```
 
-To add additional roles to the group mapping, use the following API:
+The following example request adds the role `queryRole` to the `group1map` mapping:
 
 ```
-curl -i -v  -H "Content-Type: application/json" -u internal -X POST http://localhost:8081/druid-ext/basic-security/authorization/db/ldapauth/groupMappings/group1/roles/<newrole> 
+curl -i -v  -H "Content-Type: application/json" -u internal -X POST http://localhost:8081/druid-ext/basic-security/authorization/db/ldapauth/groupMappings/group1/roles/queryrole
 ```
 
-In the next two steps you will be creating a user, and assigning previously created roles to it. These steps are only needed in the following cases: 
-                                                                                                 
- - Your LDAP server does not support the `memberOf` attribute, or 
- - You want to configure a user with additional roles that are not mapped to the group(s) that the user is a member of
- 
- If this is not the case for your scenario, you can skip these steps.
+### Step 4: Add an LDAP user to Druid and assign a role
```

Comment:
Line 216 to 231 is to map a user to role.  This section of the document is about how to map LDAP group with  druid roles hence  Line 216 to 231 is not required.

Reply:
@tijoparacka the original document for this step included the proviso - can you please confirm that this section is still not needed? What do you do if the points below apply?

You only need to complete this step if:
- Your LDAP server doesn't support the `memberOf` attribute, or
- You want to configure a user with additional Druid roles that are not mapped to the LDAP groups that the user belongs to.

Reply:
Updated after discussion with Tijo.